Rakefile

```diff
@@ -28,3 +22,10 @@ def delete_create_gradle_properties
   puts `cat #{gradle_properties_file}`
 end
 
+task :test do
```

Comment:
From the commentary, it is unclear to me why we are defining a new `test` task in this PR to override the one we pulled in from `devutils`. Were we failing to run the _gradle_ test task? Hooking into it here is new, but the commit messages or PR description don't tell me _why_.

Reply:
it's a copy-pasta convention found [elsewhere](https://github.com/logstash-plugins/logstash-input-http/blob/a36964fc007082fd7d263068e0f477e991c40656/Rakefile#L9-L15) to run both Java and Ruby tests.